CMakeLists.txt

```diff
@@ -79,6 +79,7 @@ elseif("${TARGET_ARCH}" STREQUAL "i686")
   iotjs_add_compile_flags(-D__i686__ -D__x86__ -march=i686 -m32)
 elseif("${TARGET_ARCH}" STREQUAL "x86_64")
   iotjs_add_compile_flags(-D__x86_64__)
+elseif("${TARGET_ARCH}" STREQUAL "noarch")
```

Comment:
`noarch` is quite odd for me. Can't we define a valid architecture for Rpi3? Do you want to use it for `aarch64`  or `armv8` systems?

Reply:
@LaszloLango 

AFAIK, we should not pass architecture specific options directly when we use `gbs`. `gbs` handles target specific options in its own control. If we pass architecture specific option, it would make conflicts. It's the reason @haesik stated he would like to use a common file for all architecture(https://github.com/Samsung/iotjs/pull/1385#discussion_r159364015), and I suggested `noarch` (https://github.com/Samsung/iotjs/pull/1385#discussion_r159333676) for this purpose.

If you were like me, you may feel odd about several `arm` specific options like

```cmake
set(CMAKE_SYSTEM_PROCESSOR armv7l)
```

```cmake
if("${TARGET_OS}" STREQUAL "tizen")
  iotjs_add_compile_flags(-mfpu=neon-vfpv4 -mfloat-abi=softfp)
endif()
```
The former is removed by my offline comment, but the latter still exists.

Reply:
@LaszloLango Do you need more explanation or have any siggestion?

Reply:
@glistening thanks for the explanation.